### DISCUSSION TITLE
openvslam does not work when there's a  greater distance of the camera between adjacent locations of data collection

### DISCUSSION THREAD

**Discussion by zengletian1491** [stella-cv/stella_vslam#125] [Q&A] [answered]
when I use Ricoh Theta V camera capture data and then input the data into openvslam, I find that a greater distance of the camera between adjacent locations of data collection can result in a failure processing of openvslam. Also, when the Richo Theta V rotates by a larger angle, openvslam cannot locate and map properly either. Could you give me some suggestions about how to solve this problem (when the trajectory and position of Ricoh Theta V change fast)?

**Comment by ymd-stella** [comment, marked as answer]
Increase the frame rate of the camera or decrease the speed at which the camera moves and rotates.

**Comment by dimaxano** [comment]
Also, if speak generally, the visual-inertial system will be able to handle such cases
